xdp-loader/xdp-loader.c

```diff
@@ -139,6 +148,20 @@ int do_load(const void *cfg, __unused const char *pin_root_path)
 			goto out;
 		}
 
+		if (opt->prio)
+			__xdp_program__set_run_prio(p, opt->prio);
+
+		if (opt->actions) {
+			int a;
+
+			// clear old actions
+			for (a = 0; a <= XDP_REDIRECT; a++)
+				__xdp_program__set_chain_call_enabled(p, a, 0);
```

Comment:
If you parse the option as flags, this just becomes a setter for each one instead of  this clear-then-parse-and-set dance...

Reply:
Okay, but when it becomes a settter, the obsolete chain call actions parsed from the ELF file will not be cleared. If you think this is the acceptable operation, I will remove the clear routine.